src/clr-angular/forms/select/select.spec.ts

```diff
@@ -175,5 +189,41 @@ export default function(): void {
                 expect(menus[0].classList.contains("test")).toBe(true);
             });
         });
+
+        describe("Rendering Selected Option", () => {
+            beforeEach(function() {
+                context = this.create(ClrSelect, TestOptionSelection, []);
+            });
+
+            it("renders the selected option in the input when it is clicked", () => {
+                const options = context.clarityElement.querySelectorAll(".clr-option");
+                let input: HTMLElement = context.clarityElement.querySelector(".clr-select-input");
+
+                expect(input.children.length).toBe(0);
+
+                options[0].click();
+
+                input = context.clarityElement.querySelector(".clr-select-input");
```

Comment:
Why query again, you already got the input 4 lines above?

Reply:
fixed